### ISSUE TITLE
Add slides to videos

### ISSUE THREAD

**Issue by johankj** [chriseidhof/pomotv#28]
It would be nice to have slides embedded or at least just a link to them, if they are available.

**Comment by chriseidhof** [comment]
Yeah, I think it'd be really nice to add more metadata. Slides would be awesome. Also, links in general would be cool (such as those to the Realm blog posts).
